### PR TITLE
Refresh quotes every 20 seconds

### PR DESCRIPTION
**Pull request by artursapek**
- Refresh quotes every 20 seconds
- Update UI to show when quotes are being refetched in a nicer way
- Pause quote re-fetching while user is initiating a transfer

Here's a modified version with a 5 second refresh cycle, to illustrate:

https://github.com/user-attachments/assets/c00d7665-99b9-43be-acf1-343b550c8486

### CODE REVIEW

File: wormhole-connect/src/views/v2/Bridge/ReviewTransaction/index.tsx [hunk 1]
```diff
@@ -103,16 +104,7 @@ const ReviewTransaction = (props: Props) => {
     isTransactionInProgress,
   });
 
-  const routes = useMemo(() => (route ? [route] : []), []);
-  const { quotesMap, isFetching } = useRoutesQuotesBulk(routes, {
```
Comment (artursapek) [inline wormhole-connect/src/views/v2/Bridge/ReviewTransaction/index.tsx:107]: passing this in as props now... we should have done it this way before

File: wormhole-connect/src/hooks/useRoutesQuotesBulk.ts [hunk 2]
```diff
@@ -79,6 +107,7 @@ const useRoutesQuotesBulk = (routes: string[], params: Params): HookReturn => {
     params.destToken,
     params.amount,
     params.nativeGas,
+    nonce,
```
Comment (kev1n-peters) [inline wormhole-connect/src/hooks/useRoutesQuotesBulk.ts:110]: Add `isTransactionInProgress` dependency

File: wormhole-connect/src/hooks/useRoutesQuotesBulk.ts [hunk 3]
```diff
@@ -60,16 +70,34 @@ const useRoutesQuotesBulk = (routes: string[], params: Params): HookReturn => {
...
+      const refreshTimeout = setTimeout(
+        () => setNonce(new Date().valueOf()),
+        QUOTE_REFRESH_INTERVAL,
+      );
+      setRefreshTimeout(refreshTimeout);
+    };
+
+    if (isTransactionInProgress) {
+      // Don't fetch new quotes if the user has committed to one and has initiated a transaction
+      onComplete();
```
Comment (kev1n-peters) [inline wormhole-connect/src/hooks/useRoutesQuotesBulk.ts:84]: Should we cancel the `refreshTimeout` here?

Comment (artursapek) [reply wormhole-connect/src/hooks/useRoutesQuotesBulk.ts:84]: no - all we're doing is queueing up the next iteration in 20 seconds.

File: wormhole-connect/src/hooks/useRoutesQuotesBulk.ts (hunk 2 above)
Comment (artursapek) [reply wormhole-connect/src/hooks/useRoutesQuotesBulk.ts:110]: done

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**